src/CedarCore/CoordinateHelper.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Autodesk.Revit.DB;
+
+namespace Jpp.Cedar.Core
+{
+    public static class CoordinateHelper
+    {
+        public static XYZ GetWorldCoordinates(Document document, XYZ localCoordinate)
+        {
+            if(document == null)
+                throw new System.ArgumentNullException(nameof(document));
+
+            if (localCoordinate == null)
+                throw new System.ArgumentNullException(nameof(localCoordinate));
+
+            //TODO: Ensure this method works for all 3(!) cooridnate systems
+            /*FilteredElementCollector locations = new FilteredElementCollector(document).OfClass(typeof(BasePoint));
+            foreach (var locationPoint in locations)
+            {
+                BasePoint basePoint = locationPoint as BasePoint;
+                Location svLoc = basePoint.Location;
+                double projectSurvpntX = basePoint.get_Parameter(BuiltInParameter.BASEPOINT_EASTWEST_PARAM).AsDouble();
+                double projectSurvpntY = basePoint.get_Parameter(BuiltInParameter.BASEPOINT_NORTHSOUTH_PARAM).AsDouble();
+                double projectSurvpntZ = basePoint.get_Parameter(BuiltInParameter.BASEPOINT_ELEVATION_PARAM).AsDouble();
+                //XYZ loc = (basePoint.Location as LocationPoint).Point;
+            }*/
+
+            ProjectPosition projectPosition = document.ActiveProjectLocation.get_ProjectPosition(XYZ.Zero);
```

Comment:
Don't know if the document will be required in future, if not then I'd consider passing in projectPosition.

Reply:
Will be required to get the 3(!) other coordinate systems

Reply:
Will be required to ge the 3(!) other coordinate systems